rules/integrations/endpoint/elastic_endpoint_security_behavior_detected.toml

```diff
@@ -0,0 +1,105 @@
+[metadata]
+creation_date = "2024/03/24"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.16.0"
+promotion = true
+updated_date = "2024/11/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an Elastic Defend alert for malicious behavior is received. Enabling this rule
+allows you to immediately begin investigating your Endpoint behavior alerts. This rule identifies Elastic Defend
+behavior detections only, and does not include prevention alerts.
+"""
+enabled = true
+from = "now-10m"
+index = ["logs-endpoint.alerts-*"]
+interval = "5m"
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 10000
+name = "Behavior - Detected - Elastic Defend"
+references = [
+    "https://github.com/elastic/protections-artifacts/tree/main/behavior",
+    "https://docs.elastic.co/en/integrations/endpoint",
+]
+risk_score = 47
+rule_id = "0f615fe4-eaa2-11ee-ae33-f661ea17fbce"
+rule_name_override = "message"
+setup = """
+## Setup
+
+### Elastic Defend Alerts
+This rule is designed to capture specific alerts generated by Elastic Defend.
+
+To capture all the Elastic Defend alerts, it is recommended to use all of the Elastic Defend feature-specific protection rules:
+
+Behavior - Detected - Elastic Defend (UUID: 0f615fe4-eaa2-11ee-ae33-f661ea17fbce)
+Behavior - Prevented - Elastic Defend (UUID: eb804972-ea34-11ee-a417-f661ea17fbce)
+Malicious File - Detected - Elastic Defend (UUID: f2c3caa6-ea34-11ee-a417-f661ea17fbce)
+Malicious File - Prevented - Elastic Defend (UUID: f87e6122-ea34-11ee-a417-f661ea17fbce)
+Memory Signature - Detected - Elastic Defend (UUID: 017de1e4-ea35-11ee-a417-f661ea17fbce)
+Memory Signature - Prevented - Elastic Defend (UUID: 06f3a26c-ea35-11ee-a417-f661ea17fbce)
+Ransomware - Detected - Elastic Defend (UUID: 0c74cd7e-ea35-11ee-a417-f661ea17fbce)
+Ransomware - Prevented - Elastic Defend (UUID: 10f3d520-ea35-11ee-a417-f661ea17fbce)
+
+To avoid creation of duplicate alerts, all rules mentioned above or the Elastic Defend rule (UUID: 9a1a2dae-0b5f-4c3d-8305-a268d404c306) should be enabled.
```

Comment:
```suggestion
To avoid generating duplicate alerts, you should enable either all feature-specific protection rules or the Endpoint Security (Elastic Defend) rule (UUID: 9a1a2dae-0b5f-4c3d-8305-a268d404c306).
```